### DISCUSSION TITLE
Is it possible to build kvantum for centos 7?

### DISCUSSION THREAD

**Discussion by Root-voidX** [tsujan/Kvantum#636] [Q&A]
So I am running a few commercial QT application inside a Centos 7 docker. Now the docker is using my host (fedora) kwin for windows managing but its not themeing things. I was told I would need to have the same themes installed inside the container. I wanna build kvantum standalone to just set a new qt4 and qt5 theme.

Is it possible?

**Comment by tsujan** [comment]
> Is it possible to build kvantum for centos 7

The answer to this question (the title of your comment) is "yes". CentOS is a Linux distro and you could install Kvantum on any Linux distro.

However, your comment is about Docker, not Kvantum. You might want to ask its developers.

**Reply by Root-voidX** [reply]
Which version works well for centos 7? The docker folks already says there is no limitation on their side. I'm able to run systemd and even full core load with CFD applications. I just don't know which version would work in centos with its a bit older package versions.

**Reply by tsujan** [reply]
I'm not familiar with CentOS but Kvantum 0.19.0 requires Qt 5.9.0, which should also be available in old distros. However, not having an old version of Qt here, I can't guarantee that. Kvantum is always developed against the latest Qt, although I try to support old Qt versions too.

**Reply by Root-voidX** [reply]
I see, well my centos 7 has qt version 4.8.5 but the qt5-qttools version is 5.9.7

**Reply by Root-voidX** [reply]
I built kvantum 18 and had no issues, it works, only issue now is...the damn commerical app (ansys fluent) doesn't take the qt platform theme override smh

**Reply by tsujan** [reply]
> commerical app (ansys fluent) doesn't take the qt platform theme override

Lack of platform integration may be because of a hard-coded style, which is bad practice in most cases.

**Reply by Root-voidX** [reply]
> > commerical app (ansys fluent) doesn't take the qt platform theme override
> 
> Lack of platform integration may be because of a hard-coded style, which is bad practice in most cases.

Seems like 2021 version has build in dark mode, might have to service that for uniform look. xD

But otherwise loving kvantum
